src/app/public/modules/flyout/flyout.service.ts

```diff
@@ -45,7 +50,8 @@ export class SkyFlyoutService implements OnDestroy {
   constructor(
     private adapter: SkyFlyoutAdapterService,
     private windowRef: SkyWindowRefService,
-    private dynamicComponentService: SkyDynamicComponentService
+    private dynamicComponentService: SkyDynamicComponentService,
+    private router: Router
```

Comment:
Do we also need to import the `RouterModule` into `SkyFlyoutModule`?

Reply:
It was already there :)